src/components/article/deleteArticleButton/index.js

```diff
@@ -26,7 +26,7 @@ export class DeleteButton extends Component {
 
   handleDelete(event) {
     event.preventDefault();
-    const confirmDelete = confirm("Are you sure you want to delete?");
+    const confirmDelete = window.confirm("Are you sure you want to delete?");
```

Comment:
why did you put window here??

Reply:
@wasibani-roy, `confirm()` was bringing an error in the console but when I added `window.confirm()` the error disappeared